src/frontend/src/ticketPage/Information/style.scss

```diff
@@ -0,0 +1,52 @@
+@import '../../styles/variables.scss';
+
+.information-block {
+  background-color: $dark;
+  padding: 20px;
+  margin: 20px;
+  width: 100%;
+  text-align: left;
+  height: 200px;
+  border-radius: 10px;
+  display: flex;
+  align-items: center;
+  justify-content: flex-start;
+}
+
+.time-text {
+  margin: 5px 0;
+  font-size: 20px;
+  margin-bottom: 20px;
+
+  .small {
+    margin: 5px 0;
+    font-size: 15px;
+  }  
+}
+
+.time-text-p {
+  margin: 5px 0;
+  font-size: 20px;
+  margin-top: 20px;
+}
+
+.location-text.small {
+  margin: 5px 0;
+  font-size: 15px;
+}
+
+.location-text {
```

Comment:
do you have all tags on the same level? or are they included in each other? If they are included, they need to be in one tag and included in each other according they level